mod/config/pkg/spec/testnet.go

```diff
@@ -122,6 +122,17 @@ func BaseSpec() chain.SpecData[
 		FieldElementsPerBlob:             4096,
 		BytesPerBlob:                     131072,
 		KZGCommitmentInclusionProofDepth: 17,
-		CometValues:                      cmtConsensusParams,
+		// Comet values.
+		CometValues: cmtConsensusParams,
+		// Berachain values.
+		//
+		// Expected address of the BGT contract on the bArtio testnet.
+		EVMInflationAddress: common.NewExecutionAddressFromHex(
+			"0x289274787bAF083C15A45a174b7a8e44F0720660",
+		),
```

Comment:
I don't [see this contract](https://bartio.beratrail.io/address/0x289274787bAF083C15A45a174b7a8e44F0720660) on bArtio, but its [listed in the cArtio](https://github.com/berachain/subgraphs-monorepo/blob/11530b554224ea6d3dd2448bb34369553d093e12/config/cartio.json#L27) config. Are we expecting this contract to be deployed on bArtio?

Reply:
Yes, making this more explicit. BaseSpec should be defaults, which will be used by e2e. For boonet, will set this specifically to BGT address `0x289274787bAF083C15A45a174b7a8e44F0720660`